cmd/queue/main.go

```diff
@@ -581,7 +581,7 @@ func pushRequestLogHandler(currentHandler http.Handler, env config) http.Handler
 
 func pushRequestMetricHandler(currentHandler http.Handler, countMetric *stats.Int64Measure,
 	latencyMetric *stats.Float64Measure, queueDepthMetric *stats.Int64Measure, breaker *queue.Breaker, env config) http.Handler {
-	r, err := queuestats.NewStatsReporter(env.ServingNamespace, env.ServingService, env.ServingConfiguration, env.ServingRevision, countMetric, latencyMetric, queueDepthMetric)
+	r, err := queuestats.NewStatsReporter(env.ServingNamespace, env.ServingService, env.ServingConfiguration, env.ServingRevision, env.ServingPod, countMetric, latencyMetric, queueDepthMetric)
```

Comment:
Can we change this to a struct (also it seems that 5 fields are from `env`, though `env` seems to be private for this file).

Reply:
I'll refactor it in another PR to reuse `RequestLogRevision`.